### PR TITLE
Drop stray trailing whitespace for android-project

### PR DESCRIPTION
**Pull request by janisozaur**
A simple change to drop stray trailing whitespace.

There is some leftover in src/hidapi/windows/test/data, but it looked fragile and I was afraid to touch it.

### CODE REVIEW

File: src/render/direct3d/D3D9_PixelShader_YUV.h [hunk 1]
```diff
@@ -51,114 +51,114 @@
 
 const BYTE g_ps20_main[] =
 {
-      0,   2, 255, 255, 254, 255,
```
Comment (madebr) [inline src/render/direct3d/D3D9_PixelShader_YUV.h:54]: These are auto-generated sources.

**Comment by sezero** [comment]
I don't know if we truly need this.

However, if it is to be accepted anyway:

Please drop changes to src/hidapi/* and possibly src/joystick/hidapi/steam/* :
They are 3rd party stuff

I am not sure about the worth of changes to VisualC and Xcode project files
either

Others may have more to say.

File: src/render/direct3d/D3D9_PixelShader_YUV.h (hunk 1 above)
Comment (sezero) [reply src/render/direct3d/D3D9_PixelShader_YUV.h:54]: Indeed: They should be dropped.

**Comment by icculus** [comment]
This feels unnecessary to me.

**Comment by sezero** [comment]
> This feels unnecessary to me.

Close?

**Comment by janisozaur** [comment]
I'm primarily interested in fixing the parts that are meant to be copied to other projects (e.g. the android project), but I figured I could cover the entire repository at once. I will limit the changes to the few hand-picked files.

**Comment by janisozaur** [comment]
Updated

**Comment by icculus** [comment]
Okay, sure.

**Pull request merged**